docs/refarch/boundedContexts/accountLookupAndDiscovery/index.md

```diff
@@ -44,7 +44,7 @@ Disassociate transfer/s Participants and/or Parties.
 
 #### Description
 
-Get transfer/s Participant/s details.
+Where a participant DFSP requests Participant details, this UC is used by the switch to validate the request and provide the requested relevant participant data to the requesting DFSP.
```

Comment:
```suggestion
Where a Participant DFSP requests Participant association information based on a Parties identifier.
```

Reply:
@elnyry-sam-k please review and provide your input here.

Reply:
@elnyry-sam-k & @mdebarros please let me know if you're good with this paragraph construct:

"Where a Participant DFSP requests Participant association information based on a Party identifier, this UC is used by the switch to validate the request and provide the requested participant data to the requesting DFSP."

Thanks 😄

Reply:
this is good. thank you :-)